### PR TITLE
fix: Return the whole new user created

### PR DESCRIPTION
**Pull request by ricantar**

### CODE REVIEW

File: postgres-driver/user_test.go [hunk 1]
```diff
@@ -117,13 +117,12 @@ func (ts *PGDriverTestSuite) Test_WriteNewUser() {
 		ts.Run(test.name, func() {
 			originalAccounts, err := ts.driver.ReadAccounts(context.Background(), types.DriverOptions{})
 			ts.NoError(err)
-			userID, err := ts.driver.WriteUserNewSignUp(context.Background(), test.createUser, testdata.MockTimestamp)
+			user, err := ts.driver.WriteUserNewSignUp(context.Background(), test.createUser, testdata.MockTimestamp)
+			ts.NotNil(user)
```
Comment (commoddity) [inline postgres-driver/user_test.go:121]: Do we need this? The user should be nil for the error test cases shouldn't it?

File: postgres-driver/user_test.go [hunk 2]
```diff
@@ -184,9 +183,10 @@ func (ts *PGDriverTestSuite) Test_DeletePortalUser() {
 			createdUserID := test.userID
 
 			if test.createUser.Email != "" { // if createUser set in test case use user ID from created User
-				userID, err := ts.driver.WriteUserNewSignUp(context.Background(), test.createUser, testdata.MockTimestamp)
+				user, err := ts.driver.WriteUserNewSignUp(context.Background(), test.createUser, testdata.MockTimestamp)
 				ts.Equal(test.err, err)
-				createdUserID = userID
+				ts.NotNil(user)
```
Comment (commoddity) [inline postgres-driver/user_test.go:188]: same as above comment.

**Review by commoddity — CHANGES_REQUESTED** [review]
Think we can remove those nil checks in tests right? Please correct me if I'm mistaken.

File: postgres-driver/user_test.go (hunk 1 above)
Comment (ricantar) [reply postgres-driver/user_test.go:121]: aaah you're right, over checking

**Pull request merged**